platform/default/sqlite_cache.cpp

```diff
@@ -68,7 +68,7 @@ void SQLiteCache::Impl::createSchema() {
         "CREATE TABLE IF NOT EXISTS `http_cache` ("
         "    `url` TEXT PRIMARY KEY NOT NULL,"
         "    `status` INTEGER NOT NULL," // The response status (Successful or Error).
-        "    `kind` INTEGER NOT NULL," // The kind of file.
+        "    `kind` INTEGER NOT NULL," // The kind of file. (No longer used; column remains only for backward compatibility.)
```

Comment:
You can probably remove this, as the cache will get recreated anyway after https://github.com/mapbox/mapbox-gl-native/pull/3429. Don't forget to bump the cache schema version.